cpp/game_main.cpp

```diff
@@ -345,7 +414,7 @@ bool GameMain::on_input(SDL_Event *e) {
 			break;
 		}
 		else if (not scrolling_active
-		         and e->button.button == SDL_BUTTON_MIDDLE) {
+				 and e->button.button == SDL_BUTTON_MIDDLE) {
```

Comment:
this was perfect before, have a look at https://github.com/SFTtech/openage/blob/master/doc/code_style/tabs_n_spaces.md for it :grinning: